CHANGELOG.md

```diff
@@ -4,17 +4,45 @@ Changelog for NeoFS Contract
 ## [Unreleased]
 
 ### Added
-- `verify` method for Alphabet contracts (#386)
-- script comparing some of NeoFS contract states (#399)
 
 ### Updated
-- NeoGo dependency to 0.106.0 (#398)
 
 ### Removed
 
 ### Fixed
 
-### Updating from v0.19.0
+## [0.20.0] - 2024-07-30
+
+### Added
+- `verify` method for Alphabet contracts (#386)
+- Contract deployment code (#395)
+- Functions to deal with address records and New* constructors for nns wrapper (#397)
+- Script comparing some of NeoFS contract states (#399)
+- Script to compare main/fs chain deposit state (#400)
+- Binary contracts provided as Go package (#401)
+- Support for NEP-18 addresses in NNS wrapper (#392)
+- Contract-specific constants and some types to RPC bindings (#402)
+- Prefixes to balance contract storage scheme (#406)
+- `admin` to `properties()` result of NNS (#419)
+
+### Updated
+- NeoGo dependency to 0.106.3 (#389, #398, #401, #421)
+- Contracts moved into a separate directory (#378)
+- golang.org/x/crypto dependency from 0.14.0 to 0.17.0 (#383)
+- Minimal Go version to 1.20 (#389)
+- google.golang.org/protobuf dependency from 1.31.0 to 1.33.0 (#393)
+- Licensing documentation (#391, #395, #401)
+- Release archive uses contract.nef and manifest.json file names (#401)
+- Autodeploy speedup in case Notary is available from the genesis (#410)
```

Comment:
do we need to mention it? deploy code also appears in this release, so it looks strange like we added smth and improved it in the same release

Reply:
OK, removed.